packages/plugin-session-replay-browser/README.md

```diff
@@ -51,9 +51,11 @@ const sessionReplayTracking = sessionReplayPlugin({
 
 #### Options
 
-|Name|Type|Default|Description|
-|-|-|-|-|
-|`sampleRate`|`number`|`undefined`|Use this option to control how many sessions will be selected for replay collection. A selected session will be collected for replay, while sessions that are not selected will not.  <br></br>The number should be a decimal between 0 and 1, ie `0.4`, representing the fraction of sessions you would like to have randomly selected for replay collection. Over a large number of sessions, `0.4` would select `40%` of those sessions.|
+|Name|Type|Default|Required|Description|
+|-|-|-|-|-|
+|`sampleRate`|`number`|`undefined`|Yes|Use this option to control how many sessions will be selected for replay collection. A selected session will be collected for replay, while sessions that are not selected will not.  <br></br>The number should be a decimal between 0 and 1, ie `0.4`, representing the fraction of sessions you would like to have randomly selected for replay collection. Over a large number of sessions, `0.4` would select `40%` of those sessions.|
```

Comment:
I think I remember someone saying that 40% was too much to put in the docs? Not sure if that was something here

Reply:
good point. i'll update it.